packages/spacecat-shared-rum-api-client/.nycrc.json

```diff
@@ -10,5 +10,8 @@
   "all": true,
   "include": [
     "src/**/*.js"
+  ],
+  "exclude": [
```

Comment:
why are we excluding this?

Reply:
no more exclusion. we are inclusive now ❤️